README.md

```diff
@@ -0,0 +1,281 @@
+    <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>
+    <!-- <script src="js/bootstrap.min.js"></script> -->
+    <script src="app.js" type="text/javascript"></script>
+  </body>
+</html>
+
+
+
+```
+
+and the corresponding js file(called app.js)
+
+```js
+
+function handle(event){
+
+    event.stopPropagation();
+    event.preventDefault();
+    console.log('---- handle ---');
+
+    var myform = document.getElementById('myform');
+
+    console.log('---- myform ---'+myform);
+    var payload = new FormData(myform);
+    console.log('---- handle completed---');
+
+    $.ajax({
+        type: "POST",
+        url: "YOUR FUNCTION URL HERE",
+        crossDomain: true,
+        data: payload,
+        contentType: false,
+        processData: false,
+        headers: {'x-functions-key': 'YOUR FUNCTION KEY'},
+
+        success: function (resp) {
+            console.log(resp);
+            var data = JSON.parse(resp);
+            console.log(data.result);
+
+            data = data.result[0];
+            console.log(data);
+            var emotion = data.faceAttributes.emotion;
+
+            var resultString = `
+
+            <h3>Emotions in the image:</h3><br />
+            <p>anger: ${emotion.anger}</p>
+            <p>contempt: ${emotion.contempt}</p>
+            <p>disgust: ${emotion.disgust}</p>
+            <p>fear: ${emotion.fear}</p>
+            <p>happiness: ${emotion.happiness}</p>
+            <p>neutral: ${emotion.neutral}</p>
+            <p>sadness: ${emotion.sadness}</p>
+            <p>surprise: ${emotion.surprise}</p>
+            `;
+            
+            $('#emotion').html(resultString);
+        },
+
+        error: function (err) {
+            console.log('---ERROR: '+err);
+        }
+    });
+}
+
+function clearForm(){
+    document.getElementById('myform').reset();
+    document.getElementById('emotion').innerHTML = '';
+
+}
+
+
+```
+
+The HTML contains form with a file input. Inside the app.js file, the `handle(event)`function is called upon form submission, and makes an http post call to your Azure Function with the submitted image in the body. Make sure you replace the url with your own Function url, and the key with your Function key. You can deploy this html page however you like- I'm just doing it with the **live server extension** in Visual Studio Code like so:
+
+![liveServer](images/liveServer.png)
+
+The final step for this test to work is to allow this HTML page to interact with your function through CORS. Go to your App Service resource(not the individual function) and click the CORS tab on the left side. Add your http: url to the allowed origins(this works with local hosts and live servers).
+
+![CORS](images/CORS.png)
+
+Run your HTML page, upload a file, and have fun seeing the results!
+
+
+
+![lolExample](images/lolExample.png)
+
+Check out the range in her face!! 
```

Comment:
Add a few things about what you learned from the stuff you outlined before. Did things all go to plan? What was difficult? What resources would you have wanted to have known before you started?